modules/imgproc/src/subdivision2d.cpp

```diff
@@ -210,19 +210,130 @@ void Subdiv2D::swapEdges( int edge )
     splice(sedge, getEdge(b, NEXT_AROUND_LEFT));
 }
 
-static double triangleArea( Point2f a, Point2f b, Point2f c )
-{
-    return ((double)b.x - a.x) * ((double)c.y - a.y) - ((double)b.y - a.y) * ((double)c.x - a.x);
+static double CrossProduct(Point2f u, Point2f v) {
```

Comment:
`cv::Point` and its derivatives have `cross()` method: https://docs.opencv.org/master/db/d4e/classcv_1_1Point__.html#a946769d5545dba5d328486602aae2ac3. No need to write own function.

Reply:
Fixed!